packages/react-charting/src/components/DonutChart/Arc/Arc.tsx

```diff
@@ -58,8 +58,8 @@ export class Arc extends React.Component<IArcProps, IArcState> {
           onBlur={this._onBlur}
           opacity={opacity}
           onClick={href ? this._redirectToUrl.bind(this, href) : this.props.data?.data.onClick}
-          aria-labelledby={this.props.calloutId}
-          role="text"
+          aria-label={this._getAriaLabel()}
+          role="img"
```

Comment:
Is there any downside of using img as role

Reply:
I couldn't find any. The SVG shapes are better identified by the img role and we have already used it in some of the charts.
Also, 
![image](https://user-images.githubusercontent.com/110246001/203285197-81b71b11-9057-40ed-a768-336737d1b4dd.png)